falcon/testing/helpers.py

```diff
@@ -713,7 +713,10 @@ async def _send(
 
         # NOTE(kgriffs): From the client's perspective, it was a send,
         #   but the server will be expecting websocket.receive
+        # type: Dict[str, Union[bytes, str]]
```

Comment:
Mypy says that this is invalid syntax, that's the main issue to start with I think.
Moreover, we shouldn't need type annotations in the comments now that we use `from __future__ import annotations` everywhere.